libopenage/gamestate/game_save.cpp

```diff
@@ -12,93 +14,242 @@
 #include "game_main.h"
 #include "game_save.h"
 #include "game_spec.h"
+#include "picojson.h"
```

Comment:
yeah, we are now at a point where it is possible to change the json lib we choose. So we should choose wisely.